Bottom: 23ca9bb751b7c9d5a7bd1dcef1cb836e2e56d265
Top:    00bec33d6a1ee27a3eaffcbae61740b0d0e0c800
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-04 18:51:19 +0000

Put back layer opacity to its old behaviour

The exact details of this setting aren't right yet,
so revert them so I can build a release.


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 273df0f..59ccdbd 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -1684,7 +1684,7 @@ ghid_draw_everything (BoxType *drawn_area)
   }
 
   /* draw all layers in layerstack order */
-#define FADE_FACTOR 0.8
+#define FADE_FACTOR 1
   number_phys_on_top = max_phys_group - min_phys_group;
   for (i = ngroups - 1; i >= 0; i--) {
     bool is_this_physical = drawn_groups[i] >= min_phys_group &&
